awx/main/dispatch/pool.py

```diff
@@ -132,7 +132,7 @@ def calculate_managed_tasks(self):
                 # when this occurs, it's _fine_ to ignore this KeyError because
                 # the purpose of self.managed_tasks is to just track internal
                 # state of which events are *currently* being processed.
-                pass
+                logger.info('Event UUID {} appears to be have been duplicated.'.format(uuid))
```

Comment:
Typo, maybe a `WARN`

Reply:
I suppose this is more decipherable than "UUID {} isn't UU"